src/assets/i18n/ca.json5

```diff
@@ -2558,6 +2558,7 @@
   // "error.objects": "Error fetching objects",
   "error.objects": "Error en recuperar objectes",
 
+
```

Comment:
Same here and for every other file *except* the `en.json5` file.  You should remove the unnecessary blank lines.